### PR TITLE
Implemented support for endpoint /search/stopwords (fixes #13)

### LINKED ISSUE #13: Implement support for endpoint /v1/search/stopwords
## Description
Create new method for endpoint /v1/search/stopwords

### PR DESCRIPTION
**Pull request by rui-castro**

### CODE REVIEW

**Comment by joamag** [comment]
@tsilva please review this pull request

**Comment by joamag** [comment]
@tsilva please take a look at this

File: src/farfetch/search.py [hunk 1]
```diff
@@ -63,10 +63,20 @@ def facets_product(self):
...
         url = self.base_url + "search/suggestions"
         params = dict(
-            search_terms = search_terms
+            search_terms = search_terms,
+            gender = gender
+        )
+        contents = self.get(url, params = params)
+        return contents
+
+    def search_stopwords(self, search_terms = None, gender = None):
```
Comment (tsilva) [inline src/farfetch/search.py]: search_terms is mandatory, don't put "search_terms = None"

File: src/farfetch/search.py [hunk 2]
```diff
@@ -63,10 +63,20 @@ def facets_product(self):
...
+        url = self.base_url + "search/stopwords"
+        params = dict(
+            search_terms = search_terms,
+            gender = gender
         )
         contents = self.get(url, params = params)
```
Comment (tsilva) [inline src/farfetch/search.py]: gender is optional, set the gender afterwards by doing: "if gender: params["gender"] = gender"

File: src/farfetch/search.py [hunk 3]
```diff
@@ -63,10 +63,20 @@ def facets_product(self):
         contents = self.get(url)
         return contents
 
-    def search_suggestions(self, search_terms = None):
+    def search_suggestions(self, search_terms = None, gender = None):
```
Comment (tsilva) [inline src/farfetch/search.py]: search_terms is mandatory, don't put "search_terms = None", put just "search_terms" instead

File: src/farfetch/search.py [hunk 4]
```diff
@@ -63,10 +63,20 @@ def facets_product(self):
...
 
-    def search_suggestions(self, search_terms = None):
+    def search_suggestions(self, search_terms = None, gender = None):
         url = self.base_url + "search/suggestions"
         params = dict(
-            search_terms = search_terms
+            search_terms = search_terms,
+            gender = gender
+        )
+        contents = self.get(url, params = params)
```
Comment (tsilva) [inline src/farfetch/search.py]: gender is optional, set the gender afterwards by doing: "if gender: params["gender"] = gender"

**Comment by tsilva** [comment]
@rui-castro please fix the commented stuff and ping me when done so I can merge the request

**Comment by rui-castro** [comment]
@tsilva fixed issues

**Pull request merged**